ihmc-high-level-behaviors/src/main/java/us/ihmc/behaviors/behaviorTree/BehaviorTreeNodeDefinition.java

```diff
@@ -51,8 +52,12 @@ public BehaviorTreeNodeDefinition(CRDTInfo crdtInfo, WorkspaceResourceDirectory
 
       this.saveFileDirectory = saveFileDirectory;
 
+      childrenModification = new LatestTimestampModifiable(crdtInfo);
       name = new CRDTBidirectionalString(this, BehaviorTreeDefinitionRegistry.getInitialName(getClass()));
       notes = new CRDTBidirectionalString(this, "");
+
+      setModifierName(crdtInfo.getActorDesignation().name() + ": " + name.getValue());
+      childrenModification.setModifierName(crdtInfo.getActorDesignation().name() + ": " + name.getValue() + " children");
```

Comment:
These greatly help with debugging, which need to be left in so we can debug any issues going forward.